packages/tests/web-test-runner.config.js

```diff
@@ -16,33 +18,149 @@ const removeDevModeLogging = {
   },
 };
 
-const plugins = [];
-if (process.env.TEST_PROD_BUILD) {
-  console.log("Using production builds");
-  plugins.push(fromRollup(resolveRemap)(prodResolveRemapConfig));
-} else {
-  console.log("Using development builds");
-  plugins.push(fromRollup(resolveRemap)(devResolveRemapConfig));
-  plugins.push(removeDevModeLogging);
+function getPlugins() {
+  let resolveRemapConfig;
+  if (process.env.TEST_PROD_BUILD) {
+    console.log("Using production builds");
+    resolveRemapConfig = prodResolveRemapConfig;
+  } else {
+    console.log("Using development builds");
+    resolveRemapConfig = devResolveRemapConfig;
+  }
+  return [
+    fromRollup(resolveRemap)(resolveRemapConfig),
+    removeDevModeLoggingPlugin,
+    // Detect browsers without modules (e.g. IE11) and transform to SystemJS
+    // (https://modern-web.dev/docs/dev-server/plugins/legacy/).
+    legacyPlugin,
+  ];
 }
 
+const browserPresets = {
+  // Default set of Playwright browsers to test when running locally.
+  default: ["chromium", "firefox", "webkit"],
+
+  // Browsers to test during automated continuous integration.
+  // https://saucelabs.com/platform/supported-browsers-devices
+  // https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/
+  ci: [
+    "chromium",
+    "firefox",
+    "webkit",
+    // Many browser configurations don't yet work with @web/test-runner-saucelabs.
+    // See https://github.com/modernweb-dev/web/issues/472.
+    "sauce:Windows 10/firefox@68", // Current ESR
+    //"sauce:Windows 10/chrome@latest-3", // Fails with no error message
+    //"sauce:macOS 10.15/safari@latest", // Timeout on "elements with custom properties can move between elements"
+    //"sauce:Windows 10/MicrosoftEdge@18", // Browser start timeout
+    //"sauce:Windows 7/internet explorer@11", // Browser start timeout
+  ],
+};
+
+function getBrowsers() {
+  return (process.env.BROWSERS || "preset:default")
```

Comment:
nit: use `??` instead of `||`

Reply:
I want `||` here because I want empty string to count as default.